subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -954,6 +954,10 @@ static inline void sync_info_offset_fill(struct pdu_adv_sync_info *si,
 		si->offs = offs;
 		si->offs_units = 0U;
 	}
+	/* If sync_info is part of ADV PDU the offs_adjust field
+	 * is always set to 0.
+	 */
+	si->offs_adjust = 0U;
```

Comment:
This is not needed, shall be initialized before `sync_info_offset_fill` is executed. Which you have done in `sync_info_fill`

Reply:
Thank you, I didn't spot that `fill_info_fill` is on the same execution path as `sync_info_offset_fill`. 
Line removed. I've moved the comment above the assignment into `sync_info_offset_fill`.